public/app/features/dashboard-scene/scene/DashboardSceneRenderer.tsx

```diff
@@ -22,6 +23,15 @@ export function DashboardSceneRenderer({ model }: SceneComponentProps<DashboardS
   const navModel = getNavModel(navIndex, 'dashboards/browse');
   const showDebugger = location.search.includes('scene-debugger');
 
+  if (model.isEmpty()) {
+    return (
+      <Page navModel={navModel} pageNav={pageNav} layout={PageLayoutType.Custom}>
+        <NavToolbarActions dashboard={model} />
+        <DashboardEmpty dashboard={model} canCreate={!!model.state.meta.canEdit} />
+      </Page>
+    );
+  }
```

Comment:
You can't early return this way as this will not allow accessing dashboard settings or showing the save drawer. We need to integrate this into the main render return I think.

Reply:
Showing the save drawer works, but accessing dashboard settings doesn't 🤦 I'll fix this

Reply:
Integrated this with main render. Everything seems to be working